concent_api/gatekeeper/tests/test_utils.py

```diff
@@ -0,0 +1,31 @@
+import json
+
+from django.http                    import JsonResponse
+from django.test                    import TestCase
+
+from gatekeeper.utils               import gatekeeper_access_denied_response
+
+class GatekeeperAccessDeniedResponseTest(TestCase):
+
+    def setUp(self):
```

Comment:
`full-check.sh` reports a warning for this line:

```
./concent_api/gatekeeper/tests/test_utils.py:8:1: E302 expected 2 blank lines, found 1
```

Before submitting a pull request please make sure that your code passes all tests and does not generate any new warnings.